### PR TITLE
use version from package init also for sphinx docs, insert toplevel dir ...

### PR DESCRIPTION
**Pull request by ThomasWaldmann**
...into sys.path

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -12,13 +12,28 @@
...
 # serve to show the default.
 
-import sys
+import codecs
 import os
+import re
+import sys
+
+
+def read_file(filename, encoding='utf8'):
```
Comment (kuba) [inline docs/conf.py]: that doesn't seem likely to be ever used twice in this module... inline please?

File: docs/conf.py [hunk 2]
```diff
@@ -58,9 +73,9 @@
 # built documents.
 #
 # The short X.Y version.
-version = '0.1'
+version = '.'.join(meta['version'].split('.')[:2])
```
Comment (kuba) [inline docs/conf.py]: Maybe it should be documented in `letsencrypt/__init__.py` that `__version__` must be of such form (`X.Y` at least)?

**Comment by kuba** [comment]
Thanks. Does this resolve the virtualenv issues we discussed on IRC?

**Comment by ThomasWaldmann** [comment]
as i could not clearly reproduce the issue we discussed on irc, i am not sure about this.
but as the sample config suggests to just do what i did when using autodoc, we can maybe just try.

**Comment by ThomasWaldmann** [comment]
updated.

**Comment by jdkasten** [comment]
Thanks @ThomasWaldmann!

**Pull request merged**